pkg/network/tracer/connection/tracer.go

```diff
@@ -86,29 +109,14 @@ type tracer struct {
 	// tcp_close events
 	closeConsumer *tcpCloseConsumer
 
-	pidCollisions *atomic.Int64
-	removeTuple   *netebpf.ConnTuple
-
-	telemetry telemetry
+	removeTuple *netebpf.ConnTuple
 
 	closeTracer func()
 	stopOnce    sync.Once
 
 	ebpfTracerType TracerType
-}
 
-type telemetry struct {
-	tcpConns4, tcpConns6 *atomic.Int64 `stats:""`
-	udpConns4, udpConns6 *atomic.Int64 `stats:""`
-}
-
-func newTelemetry() telemetry {
-	return telemetry{
-		tcpConns4: atomic.NewInt64(0),
-		udpConns4: atomic.NewInt64(0),
-		tcpConns6: atomic.NewInt64(0),
-		udpConns6: atomic.NewInt64(0),
-	}
+	exit chan struct{}
```

Comment:
`exitTelemetry` or something more descriptive?

Reply:
it could be used for anything that relies on the tracer closing technically. I'm open to renaming but it seems like a pattern where many types have an exit channel named `exit` or `closed` etc

Reply:
If it was used by the tracer object itself to exit, this would be ok, but it is only used by the telemetry loop, so a little more descriptive name here would help someone reading the code.